src/main/java/be/seeseemelk/mockbukkit/entity/EntityMock.java

```diff
@@ -1399,15 +1399,13 @@ public boolean isInPowderedSnow()
 	@Override
 	public boolean collidesAt(@NotNull Location location)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
```

Comment:
I think the paper implementation is has some more details to it. For example if entity is instance of armorstand and this paper config property is false `entities.armorStands.doCollisionEntityLookups`, return false. (Do we even have an paper config mock?)

I think there's some more things to this, but I honestly can't comprehend it, seems tideous to mock properly. Something something about checking against blocks and world border.

Reply:
That's correct, but at the moment we don't have a mock for _PaperConfigurations_, only for _ServerConfiguration_.
How should we address this?

Reply:
> That's correct, but at the moment we don't have a mock for _PaperConfigurations_, only for _ServerConfiguration_. How should we address this?

Let's  not address this then.